pkg/cvo/status.go

```diff
@@ -296,14 +297,17 @@ func (optr *Operator) syncStatus(ctx context.Context, original, config *configv1
 			})
 		} else {
 			var message string
+			percentComplete := float32(status.Done) / float32(status.Total)
```

Comment:
this is another `percent` that should probably be a `fraction`.

Reply:
Fixed